task01.rb

```diff
@@ -1,10 +1,12 @@
 #1. Write a script that returns true if a string contains comparison, e.g. "<23", ">45".
 
 def task01(str)
-  str.gsub!(/>>/, "")
-  str.gsub!(/<</, "")
   str.gsub!(/ /, "")
-  str.gsub!(/([<,>,=])(\d+)(\+)/) {">=" + $2}
-  str.gsub!(/([<,>,=])(\d+)(\-)/) {"<=" + $2}
-  str.include?(">") || str.include?("<")
-end
\ No newline at end of file
+  if str.scan(/[<>]=?\d+/)[0].length == str.length 
+    true
```

Comment:
@Maria If I pass "9" then this is a valid input, but has no comparison in it, so the method should return `false` instead of an error.